pom.xml

```diff
@@ -94,6 +94,47 @@
 
     <profiles>
         <profile>
+            <id>java-docs</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <version>2.10.4</version>
+                        <configuration>
+                            <aggregate>true</aggregate>
+                            <source>1.8</source>
+                            <encoding>UTF-8</encoding>
+                            <additionalparam>-Xdoclint:none</additionalparam>
+                            <sourcepath> readyapi4j-core/src/main/java:readyapi4j-core/target/generated-sources/swagger/src/main/java:readyapi4j-facade/src/main/java:readyapi4j-groovy-dsl/src/main/groovy:readyapi4j-local/src/main/java:readyapi4j-swagger/src/main/java:readyapi4j-testserver-client/src/main/java:readyapi4j-groovy-dsl/target/generated-sources/groovy-stubs/main</sourcepath>
+                            <!--<sourcepath>src/main/java;target/generated-sources/swagger/src/main/java</sourcepath>-->
```

Comment:
I wanted to make it work with relative path but couldn't. So, I kept it there to go back and dig into it sometime in future.